app/jsx/components/FilterComp.jsx

```diff
@@ -54,9 +54,9 @@ class FilterComp extends React.Component {
       <div className="c-filter">
         <h2 className="c-filter__heading">{searchString}</h2>
         <input type="hidden" name="q" value={this.props.query.q} />
-        <div>Results: {Math.ceil(this.props.count/10)} pages, {this.props.count} works</div>
+        <div>Results: {Math.ceil(this.props.count/this.props.query.rows)} pages, {this.props.count} works</div>
       {activeFilters &&
-        <FiltersShowHide activeFilters={activeFilters} handler={this.props.handler} />
+        <FiltersShowHide activeFilters={activeFilters} handler={this.props.handler} clearAll={this.clearAll} />
```

Comment:
New sub-component that will - when implemented - only show search filters when user requests that.